tests/acceptance/features/cliProvisioning/userSync.feature

```diff
@@ -82,3 +82,15 @@ Feature: sync user using occ command
     And user "regularuser" should exist
     And the display name of user "regularuser" should be "Test User"
     And the display name of user "regular" should be "Regular User"
+
+  @issue-515
+  Scenario: sync a user that does not exist
+    Given user "regularuser" should not exist
```

Comment:
I think the given step is not required

Reply:
Then how can we say the user is not created before or it doesn't exist??

Reply:
I think you can also show the command output for already existing user

Reply:
@kiranparajuli589 user0 and user1 are already created in the background, so you can also add steps to show syncing of one of them while regular user is not created we may already have some scenarios to check the existence of not created user. So checking for the existence of non-existent user is redundant.

Reply:
@HariBhandari07 I've removed the `given` step!
Please review...